build.psm1

```diff
@@ -1002,23 +1003,9 @@ function Start-PSPester {
     )
 
     $getModuleResults = Get-Module -ListAvailable -Name $Pester -ErrorAction SilentlyContinue
-    if (-not $getModuleResults)
+    if (-not ($getModuleResults | Where-Object { $_.Version -ge "4.2" } ))
```

Comment:
How about move `Where-Object` part after the `Get-Module` call? Like `Get-Module ... | Where-Object ...`. Then the if check would be simpler to read here.

Reply:
Ok, done.